cumulusci/tasks/preflight/sobjects.py

```diff
@@ -17,6 +18,48 @@ def _run_task(self):
         )
 
 
+class CheckSObjectPerms(BaseSalesforceApiTask):
+    task_options = {
+        "permissions": {
+            "description": "The object permissions to check. Each key should be an sObject API name, under which Boolean describe values can be specified, "
+            "for example, Account: queryable: True. The output is True if all sObjects and permissions are present and matching the specification.",
+            "required": True
+        }
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+
+        if type(self.options["permissions"]) is not dict:
+            raise TaskOptionsError("Each sObject should contain a map of permissions to desired values")
+
+        self.permissions = {}
+        for sobject, perms in self.options["permissions"].items():
+            self.permissions[sobject] = {perm: process_bool_arg(value) for perm, value in perms.items()}
+
+    def _run_task(self):
+        describe = {s["name"]: s for s in self.sf.describe()['sobjects']}
+
+        success = True
+
+        for sobject, perms in self.permissions.items():
+            if sobject not in describe:
+                success = False
+                self.logger.warning(f"sObject {sobject} is not present in the describe.")
+            else:
+                for perm in perms:
+                    if perm not in describe[sobject]:
+                        success = False
+                        self.logger.warning(f"Permission {perm} is not present for sObject {sobject}.")
+                    else:
+                        if describe[sobject][perm] is not perms[perm]:
+                            success = False
+                            self.logger.warning(f"Permission {perm} for sObject {sobject} is {describe[sobject][perm]}, not {perms[perm]}.")
```

Comment:
I think logging using info() would be more appropriate than warning() for all of these.